.redhat-ci.Dockerfile

```diff
@@ -14,6 +14,7 @@ RUN dnf install -y @buildsys-build && \
 
 RUN dnf install -y \
         createrepo_c \
+        /usr/bin/jq \
```

Comment:
It'll also need to be added to `.redhat-ci.yml` until the Docker image is rebuilt.